CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#8801](https://github.com/rubocop-hq/rubocop/pull/8801): Layout/SpaceAroundEqualsInParameterDefault only registered once in a line. ([@rdunlop][])
```

Comment:
Can you rebase with the latest master branch, move from `New features` section to `Bug fixes` section, and please tweak the following back ticks?

```suggestion
* [#8801](https://github.com/rubocop-hq/rubocop/pull/8801): Fix `Layout/SpaceAroundEqualsInParameterDefault` only registered once in a line. ([@rdunlop][])
```